pkg/util/cloudfoundry/garden_linux.go

```diff
@@ -0,0 +1,130 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
```

Comment:
Except if `"code.cloudfoundry.org/garden"` has some `linux` build flags, you should not need to have a `_linux` and `_windows` files as `ContainerProvider` does not depend on a specific OS anymore.